source/index.d.ts

```diff
@@ -0,0 +1,116 @@
+/// <reference types="meteor" />
```

Comment:
Do we need that in the repo? I think it should only be in the bundle, i.e., the released package. If you think it's fine, then make sure to add it to the `.eslintignore` file.

Reply:
I'm not sure if that's what you meant, but I found `/// <reference types="meteor" />` can be used here ([Docs](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#red-flags))

Reply:
I meant the default indentation of 4 spaces. And as the comment is about the entire file, I made it in the first line.